File: web/src/main/java/com/example/miagendaweb/MiAgendaWebApplication.java
package com.example.miagendaweb;

import org.springframework.boot.SpringApplication;
import org.springframework.boot.autoconfigure.SpringBootApplication;

@SpringBootApplication
public class MiAgendaWebApplication {

	public static void main(String[] args) {
		System.out.println("✅ Spring Boot arrancó correctamente");
		SpringApplication.run(MiAgendaWebApplication.class, args);
	}

}

File: web/src/main/java/com/example/miagendaweb/MiAgendaWebApplication.kt
package com.example.miagendaweb

import org.springframework.boot.SpringApplication
import org.springframework.boot.autoconfigure.SpringBootApplication

@SpringBootApplication
class MiAgendaWebApplication

fun main(args: Array<String>) {
    println("Spring Boot arranco correctamente")
    SpringApplication.run(MiAgendaWebApplication::class.java, *args)
}
